### PR TITLE
Jester Alt Title and Costume

### PR DESCRIPTION
**Pull request by Arthurdentist**
Adds Jester as an alternative title for clown and a costume to go with it.
final

### CODE REVIEW

**Comment by ComicIronic** [comment]
Can you make an alt_title switch in the clown's job code to have this stuff spawn when Jester is chosen?

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -423,4 +423,11 @@
...
-	_color = "russobluecamo"
\ No newline at end of file
+	_color = "russobluecamo"
+
+/obj/item/clothing/under/jester
+	name = "Jester Suit"
+	desc = "Only a fool would wear such a suit."
+	icon_state = "jester"
+	item_state = "jester"
+	_color = "jester"
```
Comment (ComicIronic) [inline code/modules/clothing/under/miscellaneous.dm]: EOL as well.

Comment (Arthurdentist) [inline code/modules/clothing/under/miscellaneous.dm]: EOL?

**Comment by Arthurdentist** [comment]
alt_title switch is quite a bit beyond me but I could give it a go certainly.

File: code/modules/clothing/under/miscellaneous.dm (hunk 1 above)
Comment (ComicIronic) [inline code/modules/clothing/under/miscellaneous.dm]: End of line. All files should end with an empty newline.

Comment (Arthurdentist) [inline code/modules/clothing/under/miscellaneous.dm]: alright cheers

File: code/game/jobs/job/civilian.dm [hunk 2]
```diff
@@ -250,6 +249,15 @@
...
 		H.equip_or_collect(new /obj/item/toy/waterflower(H), slot_in_backpack)
 		H.mutations.Add(M_CLUMSY)
+		if (H.mind.role_alt_title)
+			switch(H.mind.role_alt_title)
+				if("Jester")
+					H.equip_or_collect(new /obj/item/clothing/under/jester(H), slot_w_uniform)
+					H.equip_or_collect(new /obj/item/clothing/shoes/jestershoes(H), slot_shoes)
+					H.equip_or_collect(new /obj/item/clothing/head/jesterhat(H), slot_head)
+				else
+		H.equip_or_collect(new /obj/item/clothing/under/rank/clown(H), slot_w_uniform)
```
Comment (ComicIronic) [inline code/game/jobs/job/civilian.dm]: You have to indent this under the else.

**Pull request merged**